psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -3137,6 +3195,38 @@ std::vector<SharedMatrix> MintsHelper::mo_oei_deriv2(const std::string &oei_type
     return mo_grad;
 }
 
+/* Electric dipole moment derivatives in both AO and MO basis */
+
+std::vector<SharedMatrix> MintsHelper::ao_elec_dip_deriv1(int atom) {
+    std::vector<SharedMatrix> ao_grad = ao_elec_dip_deriv1_helper(atom);
+
+    return ao_grad;
+}
+
+std::vector<SharedMatrix> MintsHelper::mo_elec_dip_deriv1(int atom, SharedMatrix C1, SharedMatrix C2) {
+    std::vector<std::string> cartcomp;
+    cartcomp.push_back("X");
+    cartcomp.push_back("Y");
+    cartcomp.push_back("Z");
```

Comment:
is this a place for one of those initializer lists?

Reply:
I'm not sure I'm familiar with those. Can you expand?

Reply:
See "Initializing like arrays" example [here](https://www.geeksforgeeks.org/initialize-a-vector-in-cpp-different-ways/). It'll save you some lines of code and improve readability.